test/langPickerTest.js

```diff
@@ -110,6 +110,26 @@ describe('LanguagePicker: Pick the correct language', () => {
       ],
       expected: 'foo-Latn value',
     },
+    {
+      msg: 'Edge case: Values have value for "rm" which is also the suffix' +
```

Comment:
nit: could use a space at the end of this quoted text or the beginning in the next line

Reply:
hah, whoops :D added